### PR TITLE
US96680 - HTML editor: Save and Load description content

### PR DESCRIPTION
**Pull request by wongvincent**
- If HTML is enabled, content saves when a description editor onBlur event occurs, matching the behavior of the textarea.

- Checked that HTML editor and non-html editor still save and load descriptions (including emojis) correctly.
- Checked that when creating description using html editor, then turning html-editor off and vice versa, content still shows up correctly. In the case where html-editor was on when it was created, then turned off, we will see the tags similar to what happens when editing an html post in Activity Feed `<p>abc</p>`
- Checked that description saving fail causes failed tooltip to appear.

Bugs:
- Selecting an emoticon to insert from the popup counts as a focus-out, so it's triggering a save

### CODE REVIEW

File: editor/d2l-rubric-html-editor.html [hunk 1]
```diff
@@ -103,6 +104,9 @@
 						return 'htmleditor-' + Date.now();
 					},
 				},
+				content: {
```
Comment (mdgbayly) [inline editor/d2l-rubric-html-editor.html]: Is this `content` property being used?

Comment (mdgbayly) [reply editor/d2l-rubric-html-editor.html]: Oh - maybe I'm just not following how content get's initialized from value? It looks like it happens the other way around?

Comment (wongvincent) [reply editor/d2l-rubric-html-editor.html]: `content` is used only once to set the content tinyMCE will initially load with.
Any changes afterwards, I am saving in a `value` property.

Comment (mdgbayly) [reply editor/d2l-rubric-html-editor.html]: Oh - ok - initializing content from value was in the stuff i copied from AF.

Comment (mdgbayly) [reply editor/d2l-rubric-html-editor.html]: Not sure whether we could get away with just having a single `value` property which we use to initilize the `d2l-html-editor`'s content property, or if that causes weird issues with re-initialization when we update the value property in the blur handler?

Comment (wongvincent) [reply editor/d2l-rubric-html-editor.html]: I've tried combining the `value` and `content` property.
So the initial value we pass to the html-editor is encoded with numeric references. Whereas, when we `_getContent()`, they are character references. e.g. numeric: `&#160;`, character `&nbsp;`.

So I think it's simpler and probably clearer to just leave them as 2 separate properties. I could probably rename the property so it's clearer that the value we pass in is only used when initializing the html-editor.

**Pull request merged**